setup.cfg

```diff
@@ -66,5 +67,4 @@ install_requires =
     # we specify an unpinned Numpy which allows source distributions
     # to be used and allows wheels to be used as soon as they
     # become available.
-    numpy; python_version>='3.11'
```

Comment:
This isn't quite right, instead the version number should be changed to 3.12 here